install_base_ubuntu.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Script to install the depenencies for LISA on an Ubuntu-like system.
+
+# This is intended to be used for setting up containers and virtual machines to
+# run LISA (e.g. for CI infrastructure or for Vagrant installation). However for
+# the brave, it could be used to set up LISA directly on a real machine.
+
+usage() {
+    echo Usage: "$0" [--install-android-sdk]
+}
+
+set -eu
+
+install_android_sdk=n
+
+for arg in "$@"; do
+    if [ "$arg" == "--install-android-sdk" ]; then
+        install_android_sdk=y
+    else
+        echo "Unrecognised argument: $arg"
+        usage
+        exit 1
+    fi
+done
+
+pre_requisite_pkgs="build-essential autoconf automake libtool pkg-config trace-cmd sshpass \
+    kernelshark nmap net-tools tree python-matplotlib python-numpy libfreetype6-dev \
+    libpng12-dev python-nose python-pip python-all-dev iputils-ping git wget expect"
+
+remove_pkgs="ipython ipython-notebook"
+
+pip_pkgs="Cython trappy bart-py devlib psutil wrapt ipython jupyter"
+
+apt-get -y remove $remove_pkgs
+
+apt-get -y install $pre_requisite_pkgs
+
+yes w | pip install $pip_pkgs
+
+if [ "$install_android_sdk" == y ]; then
+    apt-get -y install openjdk-7-jre openjdk-7-jdk
+    ANDROID_SDK_URL="https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz"
+    if [ ! -e ./tools/android-sdk-linux ]; then
```

Comment:
Using this as a relative path does not work for calls like: sudo ./lisa/install_base_ubuntu.sh.